test/aia-test-srv/main.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"net/http"
+	"net/url"
+	"os"
+	"path"
+	"regexp"
+	"time"
+
+	"github.com/letsencrypt/boulder/cmd"
+	"github.com/letsencrypt/boulder/issuance"
+)
+
+type aiaTestSrv struct {
+	issuersByName map[string]*issuance.Certificate
+}
+
+func (srv *aiaTestSrv) handleIssuer(w http.ResponseWriter, r *http.Request) {
+	issuerName, err := url.PathUnescape(r.URL.Path[1:])
+	if err != nil {
+		w.WriteHeader(http.StatusBadRequest)
+		return
+	}
+
+	issuer, ok := srv.issuersByName[issuerName]
+	if !ok {
+		w.WriteHeader(http.StatusForbidden) // TODO make this StatusNotFound
```

Comment:
What needs to change before we can do this?

```suggestion
		w.WriteHeader(http.StatusForbidden) // TODO make this StatusNotFound
```

Reply:
Oops thank you, I had changed this to 403 just for debugging purposes (because I was getting 404s from somewhere else and needed to know when the request had made it to my handler) and forgot to change it back. Thanks for catching this!